grpc-ballerina/tests/grpc_tooling_common_test.bal

```diff
@@ -105,6 +105,18 @@ function testHelloWorldWithDuplicateInputOutput() {
     }
 }
 
+@test:Config {enable:true}
+function testHelloWorldChild() {
+    var result1 = assertGeneratedDataTypeSources("data-types", "child.proto", "child_pb.bal", "tool_test_data_type_14");
+    if (result1 is error) {
```

Comment:
Shall we remove the unnecessary parathesis here?

Reply:
Removed in 94d47a1